code/game/objects/items/holy_weapons.dm

```diff
@@ -316,7 +316,7 @@
 	w_class = WEIGHT_CLASS_HUGE
 	slot_flags = ITEM_SLOT_BACK|ITEM_SLOT_BELT
 	block_chance = 30
-	sharpness = IS_SHARP
+	sharpness = SHARP_EDGED
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "tore", "lacerated", "ripped", "diced", "cut")
```

Comment:
since you're refactoring a lot of this stuff, we could probably do with standardizing a few of these messages based on the sharpness of the weapon. could do it in a future pr if you don't feel like hitting that with this as well though.

Reply:
Yeah, I'll be down to chip in on a future PR for that, just don't want to bloat this one any more